emission/individual_tests/TestNominatim.py

```diff
@@ -43,24 +40,24 @@ class NominatimTest(unittest.TestCase):
 
     #Basic query to check that nominatim and geofabrik are calling the same area.
     def test_geofabrik_and_nominatim(self):
-        nominatim_result = requests.get(NOMINATIM_QUERY_URL + "/reverse?lat=41.8239891&lon=-71.4128343&format=json").json()
+        OPENSTREETMAP_result = requests.get(OPENSTREETMAP_QUERY_URL + "/reverse?lat=41.8239891&lon=-71.4128343&format=json").json()
         geofabrik_result = requests.get(GEOFABRIK_QUERY_URL + "/reverse?lat=41.8239891&lon=-71.4128343&format=json").json()
         key_list = ['osm_id', 'boundingbox']
         for k in key_list:
-            self.assertEqual(nominatim_result[k], geofabrik_result[k])
+            self.assertEqual(OPENSTREETMAP_result[k], geofabrik_result[k])
 
     #Checks the display name generated by get_filtered_place in clean_and_resample.py, which creates a cleaned place from the fake place
     # and reverse geocodes with the coordinates.
     def test_get_filtered_place(self):
-        raw_result = ecww.WrapperBase.__getattr__(clean.get_filtered_place(fake_place), "data")
-        print(NOMINATIM_QUERY_URL)
-        actual_result = ecww.WrapperBase.__getattr__(raw_result, "display_name")
+        raw_result = ecwe.Entry.__getattr__(clean.get_filtered_place(fake_place), "data")
+        print(OPENSTREETMAP_QUERY_URL)
+        actual_result = ecwe.Entry.__getattr__(raw_result, "display_name")
```

Comment:
this is not the way to use `ecwe.Entry` please look at existing usage patterns and fix

Reply:
This is addressed in `bin/createfakeplace.py`